Resources/Prototypes/_CD/Recipies/Construction/Graph/furniture.yml

```diff
@@ -0,0 +1,145 @@
+#Sofas
+- type: constructionGraph
+  id: Sofa
+  start: start
+  graph:
+    - node: start
+      actions:
+        - !type:DestroyEntity {}
+      edges:
+        - to: rattyMiddle
+          steps:
+            - material: Steel
+              amount: 2
+              doAfter: 1
+        - to: rattyRight
+          steps:
+            - material: Steel
+              amount: 2
+              doAfter: 1
+        - to: rattyLeft
+          steps:
+            - material: Steel
+              amount: 2
+              doAfter: 1
+        - to: rattyCorner
+          steps:
+            - material: Steel
+              amount: 2
+              doAfter: 1
+        - to: corpMiddle
+          steps:
+            - material: Steel
+              amount: 2
+              doAfter: 1
+        - to: corpRight
+          steps:
+            - material: Steel
+              amount: 2
+              doAfter: 1
+        - to: corpLeft
+          steps:
+            - material: Steel
+              amount: 2
+              doAfter: 1
+        - to: corpCorner
+          steps:
+            - material: Steel
+              amount: 2
+              doAfter: 1
+
+     - node: rattyMiddle
+      entity: RattySofaMiddle
+      edges:
```

Comment:
My best bet for the test fails is that line 51's `- node` is offset improperly from everything else.